src/page-support/prob-revshare/components/import.js

```diff
@@ -1,48 +1,52 @@
 import React, { useState } from 'react'
-import TextField from '@material-ui/core/TextField'
+import { TextField, Button } from '@mui/material'
 
 import { useView, useShares, ViewStates } from '../state'
-import { Button } from './button'
```

Comment:
This button component is kind of redundant so replacing it with the material UI button with the appropriate prop instead.